src/Repository/UserRepository.php

```diff
@@ -14,8 +15,23 @@
  */
 class UserRepository extends ServiceEntityRepository
 {
-    public function __construct(ManagerRegistry $registry)
+    /**
+     * @var AuthorizationCheckerInterface
+     */
+    private $authorizationChecker;
+
+    public function __construct(ManagerRegistry $registry, AuthorizationCheckerInterface $authorizationChecker)
     {
+        $this->authorizationChecker = $authorizationChecker;
         parent::__construct($registry, User::class);
     }
+
+    public function findByRole(string $role): array
+    {
+        $users = $this->findBy([], ['name' => 'ASC']);
```

Comment:
You should add an `array $orderBy` to the function parameter rather hard-coding the sorting here.

You could default to sorting by `['name' => 'ASC']`, but the signature of `findByRole` should match the other `findBy` functions.

Reply:
Added `$orderBy` parameter to the `findByRole` function.